Multichannel temperature sensor.ioc

```diff
@@ -363,8 +364,8 @@ ProjectManager.MultiThreaded=true
 ProjectManager.NoMain=false
 ProjectManager.PreviousToolchain=STM32CubeIDE
 ProjectManager.ProjectBuild=false
-ProjectManager.ProjectFileName=Multichannel temperature sensor CPP.ioc
-ProjectManager.ProjectName=Multichannel temperature sensor CPP
+ProjectManager.ProjectFileName=Multichannel temperature sensor.ioc
```

Comment:
Why do we need to change this? I have another folder called `Multichannel temperature sensor` from when I was working with C instead of C++. I'd rather keep the name with the CPP in the end, otherwise I can't use this file.